modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-fragment/selectionFilters.spec.ts

```diff
@@ -128,8 +128,12 @@ test.describe('Filters in Data Set fragment', () => {
 			await test.step(
 				'Check current items in the Frontend Data Set',
 				async () => {
+					await
+						fdsFragmentPage.fdsPaginationResults
+							.scrollIntoViewIfNeeded();
+
```

Comment:
@chestofwonders this is in case page has a lot of data, pagination is out of the viewport.

This test is a bit flaky but we'll come back to it once you fix the minor things we found in this screen